lib/src/folding/foldable_block.dart

```diff
@@ -55,38 +55,102 @@ class FoldableBlock extends InclusiveRange with EquatableMixin {
 
     return false;
   }
+
+  bool isSameLines(FoldableBlock other) {
```

Comment:
`areSameLines`?

Reply:
`is` refers to the object and not its lines: `block.is...`